robotpy_build/relink_libs.py

```diff
@@ -129,9 +123,9 @@ def relink_libs(install_root: str, pkg: PkgCfg, pkgcfg: PkgCfgProvider):
         :param pkg: Object that implements pkgcfg for this wrapper
         :param pkgcfg: robotpy-build pkgcfg resolver
     """
-    libs = {}
```

Comment:
Where is LibsDict defined? I don't see it.

Reply:
Between `set_install_name` and `_resolve_libs`.